app/templates/events/view/export.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="ui stackable very relaxed container grid">
   <div class="eight wide column">
-    {{events/view/export/download-zip}}
+    {{events/view/export/download-zip data=model startGeneration=(action 'startGeneration') eventExportStatus=eventExportStatus eventDownloadUrl=eventDownloadUrl}}
```

Comment:
@sumedh123 You need to update your tests too. How can you expect the test to work if you don't pass in an action ?